homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
```

Comment:
I think this integration is fit to move to notify entities and config sub entries. Every sub entry could be a recipient and then you can just target an entity and be done with it. It would greatly improve the user experience by getting rid of the notify service